Library/Homebrew/cmd/install.rb

```diff
@@ -98,10 +98,36 @@ def install
       else
         ofail e.message
         query = query_regexp(e.name)
-        ohai "Searching formulae..."
-        puts_columns(search_formulae(query))
+
+        ohai "Searching for similarly named formulae..."
+        formulae_search_results = search_formulae(query)
+        case formulae_search_results.length
+        when 0
+          ofail "No similarly named formulae found."
+        when 1
+          puts "This similarly named formula was found:"
+          puts_columns(formulae_search_results)
+          puts "If you meant to install it, run `brew install #{formulae_search_results.first}`"
+        else
+          puts "These similarly named formulae were found:"
+          puts_columns(formulae_search_results)
+          puts "If you meant to install one of them, run (for example) `brew install #{formulae_search_results.first}`"
```

Comment:
Here's what it looks like at 80x24.

<img width="658" alt="screen shot 2015-10-17 at 12 29 30 pm" src="https://cloud.githubusercontent.com/assets/1087646/10559989/cde74a70-74ca-11e5-911b-9337878951bf.png">

Using `e.g.` reads a bit confusing to me:

```bash
If you meant to install one of them, run e.g. `brew install homebrew/x11/atari++`
```

If the intent here is to keep the line brief, I think we can drop the "if you meant" bit:

```bash
To install one of them, run (for example) `brew install homebrew/x11/atari++`
```